src/tasks/minions/farmingActivity.ts

```diff
@@ -169,6 +172,22 @@ export default class extends Task {
 			if (!channelIsSendable(channel)) return;
 
 			channel.send(str);
+
+			if (autoFarmed) {
+				handleTripFinish(
+					this.client,
+					user,
+					channelID,
+					'',
+					res => {
+						user.log('continued trip of autofarming');
+						return this.client.commands.get('autofarm')!.run(res, []);
+					},
+					undefined,
+					data,
+					null
+				);
+			}
```

Comment:
This behaves kinda weirdly, it sends 2 messages after an autofarm now. I feel like it could be refactored a bit so a lot less code is needed, e.g.
```js
isAutofarming ? res => {
    user.log('continued trip of autofarming');
    return this.client.commands.get('autofarm')!.run(res, []);
} : undefined,
```